packages/widgets/src/dns-hole/controls/serverData.ts

```diff
@@ -0,0 +1,24 @@
+"use server";
+
+import { api } from "@homarr/api/server";
+
+import type { WidgetProps } from "../../definition";
+
+export default async function getServerDataAsync({ integrationIds }: WidgetProps<"dnsHoleControls">) {
+  const integrationId = integrationIds.at(0);
```

Comment:
Same applies here, you are going to support only one integration

Reply:
Need some help with this